src/ontology/edits/oeo-model.omn

```diff
@@ -690,6 +690,25 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
 ObjectProperty: owl:topObjectProperty
 
     
+DataProperty: OEO_00020440
+
+    Annotations: 
+        OEO_00020426 "add
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/2035
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2045",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Has scenario year value is a relation between a scenario factsheet and a xsd:dateTime value.",
```

Comment:
- I am not sure, what the exact use case of `has scenario year value` will be. But the current definition leaves open what such a relation means. So if some scenario factsheet has a year value of 2025, what is it meant to express? 
- Based on the name, should the property only care about the year value of the date-time object? If so, I think the definition should explicitly state that.

Reply:
I added a comment to make the purpose more transparent. Does that help?